io/disk/disk_info.py

```diff
@@ -151,25 +152,27 @@ def test(self):
         output = process.system_output("fdisk -l %s" % self.disk,
                                        ignore_status=True, shell=True,
                                        sudo=True)
-        if sector_string not in output:
+        if sector_string not in output.decode("utf-8"):
             msg.append("Mismatch in sector sizes of lbs,pbs in "
                        "fdisk o/p w.r.t sysfs paths")
         io_size_string = "I/O size (minimum/optimal): %s " \
                          "bytes / %s bytes" % (mis, mis)
-        if io_size_string not in output:
+        if io_size_string not in output.decode("utf-8"):
```

Comment:
Better to decode output at `process.system_output` line if it is not used for other purpose.